backend/danswer/background/celery/tasks/connector_deletion/tasks.py

```diff
@@ -81,7 +81,7 @@ def check_for_connector_deletion_task(self: Task, *, tenant_id: str | None) -> N
             "Soft time limit exceeded, task is being terminated gracefully."
         )
     except Exception:
-        task_logger.exception("Unexpected exception")
+        task_logger.exception(f"Unexpected exception: tenant={tenant_id}")
```

Comment:
Not within scope for this PR but calling out that we should likely do a rework of the background logging to more robustly include tenant IDs (similar to recent change added to API routes)